### DISCUSSION TITLE
Phase 1 deadline

### DISCUSSION THREAD

**Discussion by martinschonger** [empriselab/RCareWorld#194] [Help - Using] [answered]
Hi, I was wondering about the deadline of phase 1.
On https://emprise.cs.cornell.edu/rcareworld/challenge/ it says 'Dec 23, 2024' for phase 1.
On https://eval.ai/web/challenges/challenge-page/2351/submission it says 'Dec 2, 2024 6:00:59 AM' for phase 1.
Also on EvalAI it says 'Dec 22, 2024 6:00:00 AM CET (GMT + 1:00)' for the 'overall' challenge.
Which one is the correct end date and time? Thanks

**Comment by YoruCathy** [comment, marked as answer]
Hi Martin, it's the one on the website. We've updated the eval AI deadline as well.
